addons/ui/org.openhab.ui.cometvisu/META-INF/MANIFEST.MF

```diff
@@ -52,5 +52,4 @@ Bundle-ClassPath: .,
  lib/sqlite-jdbc-3.8.7.jar
 Export-Package: org.openhab.ui.cometvisu.php
 Bundle-Activator: org.openhab.ui.cometvisu.internal.CvActivator
- 
- 
+Bundle-ActivationPolicy: lazy
```

Comment:
@martinvw I actually only merged that PR accidentially, I only wanted to close/re-open it for the PR builder tests 😕 

The lazy activation is indeed a nasty topic. I don't fully agree with @sjka's other comment - in general, this line simply should do not difference at all (at least for bundles without an activator and for the ones with, it makes sense in most cases anyhow). And the issue is that the Eclipse IDE always shows a warning if this line is missing (and there does not seem to be a way to configure the reporting differently). So even if one could claim that it is unnecessary, it at least makes the project appear without errors/warnings in the IDE, which is imho a good thing.
@sjka, any further comments?